### DISCUSSION TITLE
I just added Worten.es, Fnac.es and Mediamarkt.es to streetmerchant

### DISCUSSION THREAD

**Discussion by Epxiahs** [jef/streetmerchant#1969] [Ideas and feature requests]
I was able to add these stores for the Spain region but I would like someone with more expertise in the area to review the code to see if something is missing, if you wanna have the stores on your SM version just hit me up.

**Comment by Borjanewsprivated** [comment]
> 
> 
> I was able to add these stores for the Spain region but I would like someone with more expertise in the area to review the code to see if something is missing, if you wanna have the stores on your SM version just hit me up.

Hi im interested in that i like buy the series x in spain

**Comment by luishg** [comment]
@Epxiahs 

I just added the Spanish version of LDLC.es to the repository for RTX products. Would be great to add your stores, so if you can send it to the repository or directly to me we can review this and add it.
